src/Generate.svelte

```diff
@@ -101,7 +101,8 @@
     seed = $reusePrompt?.seed ?? seed
     height = $reusePrompt?.height ?? height
     width = $reusePrompt?.width ?? width
-
+    useRandomSeed = false
+    copies = 1
```

Comment:
reset these to more intuitive values when a prompt is reused

Reply:
Thanks, I meant to do this.